code/orphaned procs/priority_announce.dm

```diff
@@ -27,7 +27,7 @@
 	for(var/mob/M in player_list)
 		if(!istype(M,/mob/new_player) && !M.ear_deaf)
 			M << announcement
-			if(M.client.prefs.toggles & SOUND_ANNOUNCEMENTS)
+			if(M.client.prefs.toggles & !SOUND_ANNOUNCEMENTS)
```

Comment:
It worked in my tests, I guess SOUND_ANNOUNCEMENTS will just be reversed in the code since I don't want to mess with the defaults prefs. I can also confirm that the "you will now hear announcements" text matches up with the actual behaviors